homeassistant/components/flux_led/services.yaml

```diff
@@ -0,0 +1,19 @@
+set_custom_effect:
+  description: Set a custom light effect.
+  fields:
+    entity_id:
```

Comment:
We try to use target instead now.

https://developers.home-assistant.io/docs/dev_101_services#service-descriptions

https://www.home-assistant.io/docs/blueprint/selectors/#target-selector

Reply:
I can't find any examples of where this is used in existing custom services, is this all that is required?

```
set_custom_effect:
  description: Set a custom light effect.
  target:
  fields:
    entity_id:
      description: Target light entity. (Required)
      example: 'entity_id: light.led_lights'
    colors:
      description: List of colors for the custom effect (RGB). (Required, Max 16 Colors)
      example: |
        colors:
          - [255,0,0]
          - [0,255,0]
          - [0,0,255]
    speed_pct:
      description: Effect speed for the custom effect (0-100). (Optional, default 50)
      example: 'speed_pct: 80'
    transition:
      description: Effect transition. (One of gradual, jump, strobe. Default gradual.)
      example: 'transition: jump'

```

Reply:
Here's an example:
https://github.com/home-assistant/core/blob/6023105c6a3b69ffb2338c494246802c086aaee4/homeassistant/components/wled/services.yaml#L1-L48

Note that `entity_id` is removed from `fields`.

Reply:
OK got it - thanks!